api/environmentvariables/models/env_var.go

```diff
@@ -0,0 +1,22 @@
+package models
+
+// EnvVar environment variable with metadata
+// swagger:model EnvVar
+type EnvVar struct {
+	// Name of the environment variable
+	//
+	// required: true
+	// example: VAR1
+	Name string `json:"name"`
+
+	// Value of the environment variable
+	//
+	// required: false
+	// example: value1
+	Value string `json:"value"`
+
+	// Metadata for the environment variable
+	//
+	// required: false
+	Metadata *EnvVarMetadata `json:"metadata, omitempty"`
```

Comment:
remove space in `json:"metadata, omitempty"` => `json:"metadata,omitempty"`

VS Code linter complains
![image](https://user-images.githubusercontent.com/65334626/129325295-0e5b9f52-d4e5-4274-9287-d731c4e3f016.png)

Reply:
WebStorm also indicates this - I did not notice. Fixed.